### ISSUE TITLE
Finalize text for recruiting a content strategist and post ad

### ISSUE THREAD

**Issue by itserindean** [hackforla/food-oasis#1344]
### Overview
We need to recruit a content strategist. We need an ad on github so people in the H4LA onboarding funnel make it to us.

### Action Items

- [x] find similar items from the past for same discipline even if from a different team
- [x] identify where to post this ad
- [x] write initial draft
- [x] share draft with team for review
- [ ] get reviews from other team members
     - [ ] PM
     - [ ] Design
     - [ ] Research
- [ ] write 2nd draft text based on feedback if necessary
- [ ] review and edit text
     - [ ] PM
     - [ ] Design
     - [ ] Research
- [ ] write final text based on feedback
- [ ] approve final text
     - [ ] PM
     - [ ] Design
     - [ ] Research
- [ ] post ad

**Comment by itserindean** [comment]
I found this [role description](https://github.com/hackforla/civic-opportunity/blob/master/roles-1/content-strategist-ux-writer.md)

### Content Strategist (UX Writer)
Are you a creative thinker? Do you have an interest in storytelling? If you answered yes, then HfLA is looking for you! As a Copywriter, you will have the opportunity to craft the content for the website. Collaborate with a multidisciplinary team within HfLA on creating the overall voice of the project, marketing materials, product descriptions, and direct email communications. As a Content Writer, we would rely on you to be creative, innovative, and able to easily adapt to rapid changes in stakeholder preferences.
### Responsibilities
Write clearly with the voice of the project
Collaborate with designers, PR and other professionals on large- and small-scale marketing projects (e.g. email campaigns and landing pages)
Edit and proofread copy as needed
Source images and other content needed for marketing material
### Basic Qualifications
Interest and organic knack for writing
Writing, editing and proofreading skills
Creative-minded with a sense of humor
Collaborative spirit
Willingness to keep team members informed of progress and availability
A plus if you have Experience with SEO (Search Engine Optimization) and or some knowledge of online content strategy and creation

**Comment by itserindean** [comment]
---------
### 1st Draft text 
---------
![LOGO food oasis horizontal 2](https://user-images.githubusercontent.com/107142453/186766499-656225b9-9878-4932-b4cf-3a92e6621e93.png)


**Project Name**: Food Oasis LA 
Food Oasis (http://foodoasis.la/) is a website that helps food seekers find food pantries and meal programs near them 

**Volunteer Opportunity**: We are looking for a Content Strategist (UX Writer) who can:
- Write clearly with the voice of the project
- Collaborate with designers, PR and other professionals on large- and small-scale marketing projects (e.g. email campaigns and landing pages)
- Edit and proofread copy as needed
- Source images and other content needed for marketing material
Experience is not required and anyone interested helping people having trouble finding food is welcome to apply.

**Duration**: ongoing  (~6 hrs week)

**Weekly Meeting**:
- Team meeting - Thursdays 6:00pm- 7:00pm Pacific Time

**Slack channel**: #fola
Slack name of person to contact in the channel: Erin Dean

**Read more about the project**: https://github.com/hackforla/food-oasis/wiki

**Comment by itserindean** [comment]
@fancyham @GigiUxR @staceyrebekahscott  please comment with any feedback

**Comment by GigiUxR** [comment]
Being that this is a very public facing role -- establishing online presence while building a reputation as a trusted authority to be partnered up with  / sponsored --  how certain are we that we do not want someone with at least some experience or relevant background?

**Comment by itserindean** [comment]
@GigiUxR great point. I will remove that line especially since we don't currently have anyone in a related role to  mentor.

**Comment by staceyrebekahscott** [comment]
Made a small tweak to the URL- changed http to https. 

If we are asking for someone with experience, should we specify a time period for how experienced they are, or ask for work examples?

**Comment by itserindean** [comment]
What if we work experience into the who to contact: 
Reach out to Erin Dean on #FOLA slack channel and please share any relevant experience.

**Comment by staceyrebekahscott** [comment]
@itserindean Good idea, I like that.

**Comment by staceyrebekahscott** [comment]
@itserindean A reminder that this issue is not showing on the main board because you have not added it to the project.

**Comment by fancyham** [comment]
I think the Hack for LA listing is pretty good.  Ours feels a bit too marketing-oriented in comparison?

Another possibility is that rather than having someone on FOLA staff (I'd worry there's not enough to do?), we see if we can borrow or bring on writers for project-based work from some sort of Hack for LA pool of UX writers (if there's such a thing) as needed, and that person(s) would work with us on a project basis.  For example, the writer for HfLA hired — perhaps we could borrow them once and a while? :)

Okay, so back to our listing — TL;DR: I'd suggest stealing a lot from Hack for LA's listing and adjusting it for ours :)

Also, we got a ton of recruits last time, so we should be prepared to possibly say no to multiple folks!

A little thinking out loud:

*What style do we want*
We should think out what we're looking for!  Writers can often write in completely different styles so do we want to sound authoritative?  casual?  friendly?  firm?  How do other sites write and how do we feel about them?  It would be a good exercise if we're needing to select someone.  Or perhaps this person can help us come to a style.

We should be prepared to react to their writing samples and reject if it's not a good match.  Writing is like a one-person redesign, so we should be comfortable with whatever we're putting out there.

In addition to experience, it'd be ideal to get examples of their past writing / work so that we can see if they can write for what we're looking for.


I think our current needs are more informational / functional / instructional (our static pages, automated email content/instructions) than what's in the current draft, which reads to me as more marketing-oriented (marketing projects, blog content & static web pages), so perhaps update to reflect or emphasize the former.


These are some immediate content needs off the top of my head:
- Static pages
  - About Us
  - FAQ
  - Donate/volunteer
  - Additional free food resources (that are neither pantry or meal program — perhaps links to SNAP, meals on wheels, etc. or better, to a site that lists those and ones specific to the LA area.)
  - Put Food Oasis on your website or create an instance for your region (instructions)

- Putting together a guide or set of pages specifically for LA neighborhood councils, if they're still a target (this one is marketing)

- Perhaps writing generic static page text for other regions to use (different from LA-specific text.)

- Instructional text for emails and web wizards (for our future automated emails to organizations to their own listings)


Some qualities I'd be looking for in a writer:
  - Easy to read, functional, and someone who can write to our various target audiences (food seekers, service providers, volunteers).  
  - Someone who can write for mobile (i.e. concise)
  - Bilingual in a popular language would be awesome at least for awareness.  Someone who like working with designers and researchers and is a good problem solver.

**Comment by fancyham** [comment]
Also, just sharing that writing job titles are kinda changing/ getting more refined as the industry matures:

https://uxwritinghub.com/content-strategy-and-ux-writing/

TL;DR; Content strategy used to be a too-broad blanket statement that encompassed actual corporate strategy all the way to copy writing instructions.  Now new, more specific terms Content Design, UX writing, etc. to differentiate where a person's focus is (if they have a focus), so we may start seeing those floating around.

**Comment by itserindean** [comment]
@fancyham  a couple of thoughts:

I'll make a few tweaks to remove marketing terminology and focus on our immediate needs

re: we got a ton of recruits last time. There is a designated project board for both designers and researchers and as the most recent post were the first one all onboarders saw. (And in the case of researchers were one of only 2 posts total.) There is no project board for writers/strategists likely because H4LA doesn't get a lot of said volunteers, so I'm actually worried we'll have the other problem of never getting someone.

Do you know: has H4LA hired a copywriter?

**Comment by fancyham** [comment]
> Do you know: has H4LA hired a copywriter?

I don't!  Not sure who to ask but that might be a head start or interim solution if we could borrow that person once and a while :)

**Comment by itserindean** [comment]
im not particulaly optimistic that they've have someone but i'll inquire via slack

**Comment by itserindean** [comment]
@staceyrebekahscott  yes Sam Hyler could potentially do this and she'll hopefully can help recruit 1-2 more for it.

**Comment by itserindean** [comment]
another content time is a style guide for writing...

**Comment by itserindean** [comment]
@SamHyler has joined us and helped recruit others so closing this as no longer necessary.
